packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -156,6 +156,14 @@ const double _fontSizeToScale = 14.0;
 ///
 /// {@macro flutter.widgets.RawDialogRoute}
 ///
+/// {@tool dartpad}
+/// This sample demonstrates how to create a basic date picker.
+/// When the button is tapped, [showDatePicker] is triggered and the selected date
+/// is displayed on the screen.
```

Comment:
optional suggestion:

```suggestion
/// Tapping the button calls [showDatePicker] and displays the selected date.
```

- The [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#canonical-terminology) recommends using "call" over "invoke", so perhaps it should replace "triggered" as well for consistency.
- I slightly prefer "active voice" over "passive voice".